pkgs/development/tools/misc/macdylibbundler/default.nix

```diff
@@ -11,8 +11,15 @@ stdenv.mkDerivation {
     sha256 = "149p3dcnap4hs3nhq5rfvr3m70rrb5hbr5xkj1h0gsfp0d7gvxnj";
   };
 
+  buildInputs = [ makeWrapper ];
+
   makeFlags = [ "PREFIX=$(out)" ];
 
+  postInstall = ''
+    wrapProgram $out/bin/dylibbundler \
+      --prefix PATH : "${cctools}/bin"
```

Comment:
```suggestion
      --prefix PATH ":" "${cctools}/bin"
```

Reply:
Am I accidentially getting involved in an internal style guide battle?

```
~/build/nixpkgs $ git grep -- --prefix|grep -F ' : '|wc -l
1178
~/build/nixpkgs $ git grep -- --prefix|grep -F '":"'|wc -l
197
```

:-)